src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java

```diff
@@ -0,0 +1,132 @@
+package com.solvd.micro9.users.integration.graphql;
+
+import com.solvd.micro9.users.TestUtils;
+import com.solvd.micro9.users.domain.aggregate.User;
+import com.solvd.micro9.users.domain.es.EsStatus;
+import com.solvd.micro9.users.domain.es.EsType;
+import com.solvd.micro9.users.domain.es.EsUser;
+import com.solvd.micro9.users.web.controller.GraphqlUserController;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.graphql.GraphQlTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.graphql.execution.ErrorType;
+import org.springframework.graphql.test.tester.GraphQlTester;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+@GraphQlTest(GraphqlUserController.class)
+@Import(GraphQLITConfig.class)
+public class GraphQLIT {
```

Comment:
I think you can rename this class to `GraphqlUserControllerIT` or `GraphqlUserControllerITCase`

Reply:
fixed 02fd94b